browser/css/jsdialogs.css

```diff
@@ -1759,25 +1759,49 @@ input[type='number']:hover::-webkit-outer-spin-button {
 
 /* toolbuttons */
 
-.jsdialog:not(.menubutton) > .ui-content.unobutton {
+.jsdialog:not(.menubutton)..jsdialog:not(.dropdown) {
 	width: var(--btn-size);
 }
 
-.jsdialog .ui-content.unobutton {
+.jsdialog .ui-content.unobutton.no-label.dropdown {
+	width: calc(var(--btn-size) + var(--btn-dropdown-size))
+}
+
+.jsdialog .ui-content.unobutton:not(.dropdown) {
 	height: var(--btn-size);
+	min-width: var(--btn-size);
 	padding: 0px;
 	vertical-align: middle;
 	background-color: transparent;
 	border: none;
 }
 
+.dropdown {
+	display: flex;
+}
+
```

Comment:
below `has-dropdown` class is now unused? was replaced by `dropdown`

Reply:
yes, I forgot to remove it.